hoot-test/hoot-test.pro

```diff
@@ -42,3 +42,6 @@ SOURCES += src/main/cpp/hoot/test/main.cpp \
     src/main/cpp/hoot/test/ScriptTest.cpp \
     src/main/cpp/hoot/test/ProcessPool.cpp
 
+OTHER_FILES = \
+    $$files(../test-files/cmd/*.sh, true) \
+    $$files(../scripts/core/*.sh, true) \
```

Comment:
Add the scripts as "Other Files" inside of the `hoot-test` project to make it easier to view and edit the scripts within QtCreator.